### DISCUSSION TITLE
GOL Spain

### DISCUSSION THREAD

**Discussion by pukozs75** [iptv-org/community#781] [Q&A]
### Channel Name

GOL

### Channel ID

GOLPLAY.es

### Website

https://www.goltelevision.com/en-directo

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by trancer1994** [comment]
https://d2hmr5y60yah0e.cloudfront.net/GolT_24h/golt_24h/golt_24h.m3u8 or https://play.goltelevision.com/api/manifest/live.m3u8?token=2a5a27ad0206a44d33da14da3b0277a1&expire=626f0da0

**Comment by robocopz** [comment]
> https://d2hmr5y60yah0e.cloudfront.net/GolT_24h/golt_24h/golt_24h.m3u8 or https://play.goltelevision.com/api/manifest/live.m3u8?token=2a5a27ad0206a44d33da14da3b0277a1&expire=626f0da0

Try your links now.

**Comment by AntiPontifex** [comment]
Streams do not work: please close.

**Comment by freearhey** [comment]
https://d2hmr5y60yah0e.cloudfront.net/GolT_24h/golt_24h/golt_24h.m3u8 (offline)
<img width="530" alt="image" src="https://user-images.githubusercontent.com/7253922/232227202-6a073039-7c44-4059-aa5f-a7b390ac0ed1.png">

https://play.goltelevision.com/api/manifest/live.m3u8?token=2a5a27ad0206a44d33da14da3b0277a1&expire=626f0da0 (offline)
<img width="814" alt="image" src="https://user-images.githubusercontent.com/7253922/232227239-fc23768f-f791-4f99-8d04-7b266a9eef77.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/es.m3u#L284-L285
